tests/fixtures/controllers/methodController.ts

```diff
@@ -85,11 +86,8 @@ export class MethodController extends Controller {
   public async oauthAndAPIkeySecurity(): Promise<TestModel> {
     return new ModelService().getModel();
   }
-
-  /**
-   * @deprecated
```

Comment:
Please revert the changes to this file since they test existing functionality that can't be removed.

Reply:
I just reverted these changes